site/content/en/latest/tasks/security/jwt-authentication.md

```diff
@@ -150,10 +150,13 @@ You should see the below response
 
 ## Connect to a remote JWKS with Self-Signed Certificate
 
-To connect to a remote JWKS with a self-signed certificate, you need to configure it using the [Backend] resource within the [SecurityPolicy]. Additionally, use the [BackendTLSPolicy] to specify the CA certificate required to authenticate the JWKS host.
+To connect to a remote JWKS with a self-signed certificate, you need to configure it using the [BackendTLSPolicy] to specify the CA certificate required to authenticate the JWKS host. Additionally, if the JWKS is outside of the cluster, you need to configure the [Backend] resource to specify the JWKS host.
 
 The following example demonstrates how to configure the remote JWKS with a self-signed certificate.
 
+Please note that the `Backend` resource is not required if the JWKS is hosted on the same cluster as the Envoy Gateway, since
+it can be accessed directly via the Kubernetes Service DNS name.
```

Comment:
Update the existing docs for Self-Signed Certificate: `Backend` resource is only required for out-of-cluster JWKS.